src/features/user/LoginOverlay.vue

```diff
@@ -0,0 +1,176 @@
+<template>
+  <SimpleSpinner v-if="currentUserState.loading" />
+  <template v-else>
+    <slot v-if="initialized" :onLogout="onLogout"></slot>
+    <dialog ref="dialogRef" @keyup.escape.prevent @keydown.escape.prevent>
+      <a
+        :href="redirectUrl"
+        target="_blank"
+        @click="onLinkClick"
+        @keydown.enter="onLinkClick"
+        >Uw sessie is verlopen. Klik in het scherm om opnieuw in te loggen. Als
+        u dit binnen {{ timer }} seconden doet verliest u geen werk</a
```

Comment:
dat aftellen is erg grappig, maar de hij maakt rare sprongen en begin af en toe overnieuw. uiteindelijk ben je wel na ongeveer een minuut uitgelogd. (getest door in een tweede tab uit te loggen en dan terug te gaan naar de eerste tab)
misschien gewoon niet de teller tonen, maar alleen een statische tekst?

Reply:
Ja mee eens. Leuk idee, maar als het niet werkt liever simpel.

Reply:
Werkt nu idd met een statische tekst